client/src/common/components/Avatar.jsx

```diff
@@ -1,9 +1,12 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { injectIntl } from 'react-intl';
```

Comment:
hmm `Avatar` is a simple component to only renders an image. It gets a user name from props passed by its parent. Could we pass already `formattedName` to  `Avatar` and not inject additional libs/logic to it?